ttnn/cpp/ttnn/decorators.hpp

```diff
@@ -9,8 +9,12 @@
 #include "tt_metal/third_party/tracy/public/tracy/Tracy.hpp"
 #include "ttnn/core.hpp"
 #include "ttnn/operation.hpp"
+#include "ttnn/operation.hpp"
+#include "ttnn/device_operation.hpp"
 #include "ttnn/run_operation.hpp"
 #include "ttnn/tensor/tensor.hpp"
+#include "ttnn/common/constants.hpp"
+#include "ttnn/device_operation.hpp"
```

Comment:
More includes make this header 2x and it is included all over the place and is one of the reasons for longer build time. Need to keep this in mind

Reply:
I know. But this will now be the only place where ""ttnn/device_operation.hpp" is needed. I will add another PR where I remove it and its functions from being used by ops directly